src/networks.ts

```diff
@@ -1103,17 +1231,38 @@ export const networks: Networks = {
     key: 'ethereumTestnetGoerli',
     shortKey: 'goerli',
     color: '#83EEFF',
-    chain: 5,
+    chain: 11155111,
```

Comment:
@alexanderattar I think this is wrong right? i think the wrong field got modified.